platform/app/src/routes/index.tsx

```diff
@@ -112,6 +113,7 @@ const createRoutes = ({
   const allRoutes = [
     ...routes,
     ...(showStudyList ? [WorkListRoute] : []),
+    ...(publicUrl !== '/' && showStudyList ? [{ ...WorkListRoute, path: publicUrl }] : []),
```

Comment:
Do we need the line before? since that has also `...(showStudyList ? [WorkListRoute] : []),`

Reply:
Yes, the first one is on the / and the second on the public url.  I swapped those so the second is on the public url.